src/frequenz/sdk/timeseries/_formula_engine/_formula_engine_pool.py

```diff
@@ -110,3 +110,8 @@ def from_generator(
         ).generate()
```

Comment:
I found the commit `Add methods for stopping running formula engines` is misleading because the changes in the commit are not only adding methods for stopping running formula engines but also updating existing ones.
So you might want to clarify the motivation or reason to update the existing methods or split those changes into a separate commit